test/common.test.ts

```diff
@@ -46,7 +47,17 @@ describe("Utils test", () => {
     const provider = new providers.JsonRpcProvider(rpcUrl, 1);
     const spokePool: SpokePool = SpokePool__factory.connect(spokePoolAddress, provider);
 
-    const unsignedTxn = await createUnsignedFillRelayTransaction(spokePool, usdcAddress, relayerAddress);
+    const fill: Fill = buildFillForSimulatingFullDeposit(
```

Comment:
Should you add a test that this conditional `toBN(relayerBalanceOfToken).lt(toBN(amountToRelay))` throws an error?